app/index.js

```diff
@@ -43,12 +44,17 @@ bot.on('message', (msg) => {
                 return daily(body, bot, chatId)
             }
 
-            const pattern = /daily (\d+)/
-            if (pattern.test(userMessage)) {
-                const n = pattern.exec(userMessage)[1]
+            const dailyPattern = /daily (\d+)/
+            if (dailyPattern.test(userMessage)) {
+                const n = dailyPattern.exec(userMessage)[1]
                 return daily(body, bot, chatId, n)
             }
 
+            const testingDefaultPattern = /test( ?[a-zA-Z&])+/
```

Comment:
Please use `test ( ?[a-zA-Z&])+`. As the current one accepts `testxyz hello` also.

Reply:
Yup. On it.